src/common/meta/src/key/datanode_table.rs

```diff
@@ -38,14 +38,18 @@ use crate::DatanodeId;
 /// For compatible reason, DON'T modify the field name.
 pub struct RegionInfo {
     #[serde(default)]
-    // The table engine, it SHOULD be immutable after created.
+    /// The table engine, it SHOULD be immutable after created.
     pub engine: String,
-    // The region storage path, it SHOULD be immutable after created.
+    /// The region storage path, it SHOULD be immutable after created.
     #[serde(default)]
     pub region_storage_path: String,
-    // The region options.
+    /// The region options.
     #[serde(default)]
     pub region_options: HashMap<String, String>,
+    /// The per-region wal options.
+    /// Key: region number (in string representation). Value: the encoded wal options of the region.
```

Comment:
I am sorry, i want to know why we use string for region number here.

Reply:
See the test: 
https://github.com/GreptimeTeam/greptimedb/blob/bf2e93d704be1ef7148bb15863eb5e09aa25818e/src/common/meta/src/key/datanode_table.rs#L305-L338

Reply:
If the region number is of type u32, `serde_json::to_vec` + `serde_json::from_slice` would panic.